core/core-funcs.el

```diff
@@ -119,6 +119,14 @@ Supported properties:
   (re-search-forward anchor-text)
   (beginning-of-line)
   (show-subtree)
+  (setq-local org-emphasis-alist '(("*" bold)
+                                   ("/" italic)
+                                   ("_" underline)
+                                   ("=" org-verbatim verbatim)
+                                   ("~" org-kbd)
+                                   ("+"
+                                    (:strike-through t))))
+
```

Comment:
Can you just push to the alist the ("~" org-kbd) ? I have stuff in my configuration that pushes to this same alist to have a face defined for :strike-through as well and this will break it ;_;. Also, what is different between setq-local and plain setq here?